engine/NetworkProcessorNet.go

```diff
@@ -103,7 +104,7 @@ func Peers(fnode *FactomNode) {
 			case constants.MISSING_DATA:
 				if !fnode.State.DBFinished {
 					return true
-				} else if fnode.State.InMsgQueue().Length() > constants.INMSGQUEUE_HIGH {
+				} else if fnode.State.DataMsgQueue().Length() > constants.INMSGQUEUE_HIGH { // REVIEW: should this limit be altered?
```

Comment:
@factom-clay do you think this same limit is OK to use here?

Reply:
In general, unless it's required to prevent deadlock, I'm against dropping messages.